mlabs/efficient-nft-pab/Main.hs

```diff
@@ -70,23 +69,16 @@ mkPolicy oref _ ctx =
 mkPolicyUntyped :: BuiltinData -> BuiltinData -> ScriptContext -> Bool
 mkPolicyUntyped oref = mkPolicy (unsafeFromBuiltinData oref)
 
-
 policyUntyped :: TxOutRef -> MintingPolicy
 policyUntyped oref =
   mkMintingPolicyScript $
-    $$(PlutusTx.compile [|| (\a -> wrapMintingPolicy (mkPolicyUntyped a)) ||])
-     `PlutusTx.applyCode` PlutusTx.liftCode (PlutusTx.toBuiltinData oref)
-  -- MintingPolicy $
-  --   policyUntypedScript
-  --     `applyArguments` [ toData $ getTxId $ txOutRefId oref,
-  --                        toData $ txOutRefIdx oref
-  --                      ]
+    $$(PlutusTx.compile [||(wrapMintingPolicy . mkPolicyUntyped)||])
+      `PlutusTx.applyCode` PlutusTx.liftCode (PlutusTx.toBuiltinData oref)
 
 policyUntypedScript :: Script
 policyUntypedScript =
   fromCompiledCode
-    $$(PlutusTx.compile [||(\a -> wrapMintingPolicy (mkPolicyUntyped a))||])
-
+    $$(PlutusTx.compile [||(wrapMintingPolicy . mkPolicyUntyped)||])
```

Comment:
This actually slightly changes the encoded script from what I have been testing with, I'll make the update and test again just to be sure.

Reply:
Tested as described in https://github.com/mlabs-haskell/seabug-contracts/issues/24, and also used it to successfully mint :+1: